hocuspocus-extension-elasticsearch/src/elasticsearch.ts

```diff
@@ -39,18 +41,38 @@ export class Elasticsearch extends Database {
         return null;
       }
     },
-    store: async ({ documentName, state }) => {
+    store: async ({ documentName, state, document }) => {
       // console.log(`DB store ${state}`)
       await this.db?.update({
```

Comment:
nit: As the updated document and inserted document are the same, we can use `index` operation instead, which create or update the entire document. At least we don't need to repeat `ydoc` twice.
https://www.elastic.co/guide/en/elasticsearch/reference/6.8/docs-index_.html